test/e2e-v2/cases/storage/storage-cases.yaml

```diff
@@ -47,7 +47,7 @@ cases:
   - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql trace ls
     expected: expected/traces-list.yml
   # negative tags search: relationship should be logical AND instead of logical OR
-  - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql trace ls --tags http.method=POST,http.status_code=200
+  - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql trace ls --tags http.method=POST,http.status_code=200,iserror=true
```

Comment:
Why do you change this case?

Reply:
So? What is wrong? e2e locked the agent version, and the PR you posted has been merged months ago.
e2e is not for every released version, that is not the purpose.

Reply:
Before  #258 PR , All http plugins only  collect http code which max 400 , so   the case was meet .
But  since #285 , http plugins collect all  http code , and  I updated `SW_AGENT_JAVA_COMMIT` to latest , so  the the case couldn't meet . In  order keep it correct , I added the param 


BTW  the expected data of this case if  `traces: []` .

Reply:
Then, why do you use a meaningless tag? Such as `randomKey=randomValue`? `isError` is a segment field, rather than a tag. Using this similar but not the same tag key would make ppl confused.

https://github.com/apache/skywalking-query-protocol/blob/2c541beae657306127f07af127d179492a1d7cf9/trace.graphqls#L28

Reply:
OK, I would choose an appropriate tag